packages/expo-dev-launcher/bundle/screens/CrashReportScreen.tsx

```diff
@@ -0,0 +1,107 @@
+import { View, Text, Heading, Button, Row, Spacer } from 'expo-dev-client-components';
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+
+import { useBuildInfo } from '../hooks/useBuildInfo';
+import { copyToClipboardAsync } from '../native-modules/DevLauncherInternal';
+
+export function CrashReportScreen({ route }) {
+  const [clipboardError, setClipboardError] = React.useState('');
+  const [clipboardContent, setClipboardContent] = React.useState('');
+
+  const buildInfo = useBuildInfo();
+
+  const { message, stack, timestamp } = route.params;
+  const date = new Date(timestamp).toUTCString();
+
+  const timerRef = React.useRef(null);
+
+  React.useEffect(() => {
+    if (clipboardContent) {
+      timerRef.current = setTimeout(() => {
+        setClipboardContent('');
+      }, 3000);
+    }
+
+    return () => {
+      clearTimeout(timerRef.current);
+    };
+  }, [clipboardContent]);
+
+  const onCopyPress = async () => {
+    const { runtimeVersion, sdkVersion, appName, appVersion } = buildInfo;
+
+    const content = JSON.stringify(
+      {
+        appName,
+        timestamp,
+        message,
+        runtimeVersion,
+        sdkVersion,
+        appVersion,
+        stack,
+      },
+      null,
+      '\t'
```

Comment:
```suggestion
      2
```
to match what we do in dev-menu https://github.com/expo/expo/blob/f5787adbce4bc4b56e730a8938797d618a3247d8/packages/expo-dev-menu/app/hooks/useClipboard.tsx#L26